src/main.c

```diff
@@ -20,12 +20,17 @@
 
 extern void SetSysClock(bool overclock);
 
+serialPort_t *Serial1;
+
 int main(void)
 {
   // Configure clock, this figures out HSE for hardware autodetect
   SetSysClock(0);
   systemInit();
 
+  // Initialize Serial ports
+  Serial1 = uartOpen(USART1, NULL, 921600, MODE_RXTX);
```

Comment:
So it looks this removes support for setting the baud rate via a param. Just checking, is that what you intended?